$NetBSD: patch-utils_docbook2odf,v 1.1 2017/03/15 18:43:41 bsiegert Exp $

Correct confusion with "require" vs. "use" capabilities, otherwise
the test returns false.

--- utils/docbook2odf.orig	2007-05-19 06:55:08.000000000 -0400
+++ utils/docbook2odf
@@ -49,7 +49,7 @@
 }
 
 my $USE_ZIP = 0;
-if (eval "require Archive::Zip qw( :ERROR_CODES :CONSTANTS )")
+if (eval "require Archive::Zip;")
 {
 	$USE_ZIP = 1;
 }
